people/am3458.md

```diff
@@ -0,0 +1,8 @@
+Amanda Moy is a first-year DBMI PhD with an interest in pubic health informatics.
+---
+layout: people
```

Comment:
I believe this front matter needs to be at the very top of the page

Reply:
Updated